### DISCUSSION TITLE
[일지] 5월 25일 (화) 오후 회의록

### DISCUSSION THREAD

**Discussion by olenmg** [bcaitech1/p4-opt-5-vibrhanium-#7] [일지]
## 5월 25일(화) 오후 회의록

반말 시작! 

- Q: 기존 만든 깃헙은?
- A: 사용 안하고 부스트캠프 [github](https://github.com/bcaitech1/p4-opt-5-vibrhanium-)만 사용할 예정

- Q: Discussion과 Issue의 차이점?
- A: Discussion이 더 자유로운 분위기. 여기서 토의를 통해 확정된 내용을 Issue에 등록하여 처리.

### 규칙
- 찾은 자료는 Slack 자료 채널에 올린 후 => 피어세션에서 얘기해본 이후 => 우리에게 도움이 되는 자료면 Discussion - 자료에 올리기

### 자료 공유
"어디까지 깎아봤니?: 모바일 서비스를 위한 가벼운 이미지 인식/검출용 딥러닝 모델 설계"재영언니 insight 
- pretrained model 사용하기
- 최신 가벼운 모델들 실험해보기(MobileNetV1, MobileNetV2, MobileNetV3,  ShuffleNetV1, ShuffleNetV2)

### 아이디어
- 이미지를 잘라서, 데이터를 augmentation한 후 이걸로 학습
    - competition에서의 MACs는 감소
- MACs는 모델 사이즈와 이미지 사이즈에만 영향을 받는다.
- fc layer 를 전체 conv로? 찾아보기 

### 할 것 
- wandb ID 팀 파기 (아름)
  - 종헌: olenmg
  - 건우: afnf33
  - 명환: hwan17
  - 재경: jaegyeong
  - 재영: joielee09

- Validation/Test 분포(성능)가 비슷하도록 validation set 뽑기 (건우)
    https://www.kaggle.com/c/LANL-Earthquake-Prediction/discussion/94390
    - 분포가 train이랑 비슷하다
        - stratified Kfold => 1개만 뽑아서 활용
    - 100/50/10/70/... =>  10/5/1/7/...
    - 10/10/10/10
    - 분포가 전체가 일정하다(30/30/30 ...)
        - random으로 일정한 개수만큼씩 뽑아오기 => 제출 3개
    - 목표: 신뢰도 높은 validation set을 뽑아내자.
    - + 분포 정보

- Daily mission 연장선에서 EDA 더 한 것 (재경)
    - EDA 예시
        - augmentation 했을 때 사람이 봐도 구별할 수 있는건지?
        - 휴리스틱하게 이게 의미있는 이미지인지 직접 구별
        - 아웃라이어 확인
        - 밝기 확인
        - Category별 이미지 크기 확인

- validation에서 어떤 것들을 잘 맞히는지, 어떤 것들을 위주로 많이 틀리는지 확인하는 코드작성 (명환)

- submission 파일 결과 확인 하는 코드 (아름)

- Augmentation 방법 찾기 (종헌)

- basecode에 wandb 세팅추가하기 (명환)

- 적절한 모델 찾기(재영)
